apcd-cms/src/apps/admin_exception/urls.py

```diff
@@ -8,11 +8,11 @@
     path('list-exceptions/<str:status>', AdminExceptionsTable.as_view(), name='status'),
     path('list-exceptions/<str:org>', AdminExceptionsTable.as_view(), name='org'),
     path('list-exceptions/<str:status><str:org>', AdminExceptionsTable.as_view(), name='status_org'),
-    path(r'list-exceptions/api/', AdminExceptionsTable.as_view(), name='admin_exceptions_table_api'),
-    path(r'list-exceptions/api/?status=(?P<status>)/', AdminExceptionsTable.as_view(),
+    path('list-exceptions/api/', AdminExceptionsTable.as_view(), name='admin_exceptions_table_api'),
+    path('list-exceptions/api/?status=<status>/', AdminExceptionsTable.as_view(),
          name='admin_exceptions_table_api'),
-    path(r'list-exceptions/api/?org=(?P<org>)/', AdminExceptionsTable.as_view(), name='admin_exceptions_table_api'),
-    path(r'list-exceptions/api/?status=(?P<status>)&org=(?P<org>)/', AdminExceptionsTable.as_view(),
+    path('list-exceptions/api/?org=<org>/', AdminExceptionsTable.as_view(), name='admin_exceptions_table_api'),
+    path('list-exceptions/api/?status=<status>&org=<org>/', AdminExceptionsTable.as_view(),
          name='admin_exceptions_table_api'),
```

Comment:
All of this can be replaced with and no need for regex usage. The underlying view is doing this `status_filter = self.request.GET.get('status')`
```
    path('list-exceptions/api/', AdminExceptionsTable.as_view(), name='admin_exceptions_table_api'),
    path('exceptions/<int:exception_id>/', UpdateExceptionView.as_view(), name='update_exceptions'),
```